crates/matrix-sdk/src/docs/sync_running.md

```diff
@@ -1 +1,20 @@
-This method requires a running sync to complete.
\ No newline at end of file
+## Synced State
+This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires a running sync to complete. This functionality is implemented for:
++ [Client::create_room][client_create_room]
++ [Client::join_room_by_id][client_join]
++ [Client::join_room_by_id_or_alias][client_join_alias]
++ [Invited::accept_invitation][invited_accept]
++ [Left::join][left_join]
```

Comment:
I was actually thinking of links to the various sync methods 😅

Reply:
riiight 🥲